containers/runner/launch

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I wonder if Pitti was using /bin/sh for some other reason than being as common as possible, but I think the change is OK.